SignTool Error

I've encountered the following error while trying to use this action to sign the binaries. It seems like it's usually related to spaces in the path to certificate, but there are no spaces in this case.

Here's how I am [using](https://github.com/trollixx/zeal/blob/main/.github/workflows/build-check.yml#L158-L168) it:

```yaml
      - name: Sign Artifacts
        uses: dlemstra/code-sign-action@v1
        with:
          certificate: '${{ secrets.CODESIGN_CERTIFICATE }}'
          password: '${{ secrets.CODESIGN_PASSWORD }}'
          files: |
            build/${{ matrix.config.configurePreset }}/bin/RelWithDebInfo/zeal.exe
            build/${{ matrix.config.configurePreset }}/bin/RelWithDebInfo/archive.dll
            build/${{ matrix.config.configurePreset }}/bin/RelWithDebInfo/sqlite3.dll
            build/${{ matrix.config.configurePreset }}/bin/RelWithDebInfo/zlib1.dll
```

Action log:

```
2023-04-29T20:38:13.6260960Z ##[group]Run dlemstra/code-sign-action@v1
2023-04-29T20:38:13.6262050Z with:
2023-04-29T20:38:13.6339947Z   certificate: ***

2023-04-29T20:38:13.6340969Z   password: ***
2023-04-29T20:38:13.6352367Z   files: build/ninja-multi-vcpkg/bin/RelWithDebInfo/zeal.exe
build/ninja-multi-vcpkg/bin/RelWithDebInfo/archive.dll
build/ninja-multi-vcpkg/bin/RelWithDebInfo/sqlite3.dll
build/ninja-multi-vcpkg/bin/RelWithDebInfo/zlib1.dll

2023-04-29T20:38:13.6354242Z env:
2023-04-29T20:38:13.6355170Z   VCPKG_DEFAULT_TRIPLET: x64-windows-release
2023-04-29T20:38:13.6356276Z   VCPKG_DEFAULT_BINARY_CACHE: D:/a/zeal/b/vcpkg_cache
2023-04-29T20:38:13.6357438Z   VCPKG_BINARY_SOURCES: clear;x-gha,readwrite
2023-04-29T20:38:13.6359162Z   ACTIONS_CACHE_URL: https://artifactcache.actions.githubusercontent.com/khSvqTGLg4ru7uso17fKDw0EmHh2uBnuQiKush7LG516fzTzkm/
2023-04-29T20:38:13.6405607Z   ACTIONS_RUNTIME_TOKEN: ***
2023-04-29T20:38:13.6406535Z   RUNVCPKG_VCPKG_ROOT: D:\a\zeal\vcpkg
2023-04-29T20:38:13.6407547Z   VCPKG_ROOT: D:\a\zeal\vcpkg
2023-04-29T20:38:13.6408579Z   pythonLocation: C:\hostedtoolcache\windows\Python\3.11.3\x64
2023-04-29T20:38:13.6409987Z   PKG_CONFIG_PATH: C:\hostedtoolcache\windows\Python\3.11.3\x64/lib/pkgconfig
2023-04-29T20:38:13.6411346Z   Python_ROOT_DIR: C:\hostedtoolcache\windows\Python\3.11.3\x64
2023-04-29T20:38:13.6412635Z   Python2_ROOT_DIR: C:\hostedtoolcache\windows\Python\3.11.3\x64
2023-04-29T20:38:13.6413928Z   Python3_ROOT_DIR: C:\hostedtoolcache\windows\Python\3.11.3\x64
2023-04-29T20:38:13.6415155Z   Qt6_DIR: D:\a\zeal\Qt\6.5.0\msvc2019_64
2023-04-29T20:38:13.6416205Z   QT_PLUGIN_PATH: D:\a\zeal\Qt\6.5.0\msvc2019_64\plugins
2023-04-29T20:38:13.6417373Z   QML2_IMPORT_PATH: D:\a\zeal\Qt\6.5.0\msvc2019_64\qml
2023-04-29T20:38:13.6418450Z   CC: cl.exe
2023-04-29T20:38:13.6419295Z   CXX: cl.exe
2023-04-29T20:38:13.6420177Z   ZEAL_VERSION: 0.6.2-dev.299+fcab16b
2023-04-29T20:38:13.6421137Z ##[endgroup]
2023-04-29T20:38:13.7328051Z Writing 8553 bytes to C:\Users\RUNNER~1\AppData\Local\Temp\code-sign-certificate-497841.pfx.
2023-04-29T20:38:14.0977310Z Signtool location is C:/Program Files (x86)/Windows Kits/10/bin/10.0.22621.0/x86/signtool.exe.
2023-04-29T20:38:14.0980401Z Signing: build/ninja-multi-vcpkg/bin/RelWithDebInfo/zeal.exe.
2023-04-29T20:38:16.2296681Z 
2023-04-29T20:38:16.2298035Z SignTool Error: An error occurred while attempting to load the signing
2023-04-29T20:38:16.2299088Z 
2023-04-29T20:38:16.2300424Z 	certificate from: C:\Users\RUNNER~1\AppData\Local\Temp\code-sign-certificate-497841.pfx
2023-04-29T20:38:16.2301410Z 
2023-04-29T20:38:16.2301434Z 
2023-04-29T20:38:16.2301879Z Waiting for 1 seconds.
2023-04-29T20:38:17.2819050Z 
2023-04-29T20:38:17.2820397Z SignTool Error: An error occurred while attempting to load the signing
2023-04-29T20:38:17.2821310Z 
2023-04-29T20:38:17.2822595Z 	certificate from: C:\Users\RUNNER~1\AppData\Local\Temp\code-sign-certificate-497841.pfx
2023-04-29T20:38:17.2823573Z 
2023-04-29T20:38:17.2823588Z 
2023-04-29T20:38:17.2824049Z Waiting for 2 seconds.
2023-04-29T20:38:19.3278335Z 
2023-04-29T20:38:19.3279682Z SignTool Error: An error occurred while attempting to load the signing
2023-04-29T20:38:19.3280765Z 
2023-04-29T20:38:19.3282501Z 	certificate from: C:\Users\RUNNER~1\AppData\Local\Temp\code-sign-certificate-497841.pfx
2023-04-29T20:38:19.3283541Z 
2023-04-29T20:38:19.3283552Z 
2023-04-29T20:38:19.3284020Z Waiting for 3 seconds.
2023-04-29T20:38:22.3772912Z 
2023-04-29T20:38:22.3774242Z SignTool Error: An error occurred while attempting to load the signing
2023-04-29T20:38:22.3775313Z 
2023-04-29T20:38:22.3776646Z 	certificate from: C:\Users\RUNNER~1\AppData\Local\Temp\code-sign-certificate-497841.pfx
2023-04-29T20:38:22.3777650Z 
2023-04-29T20:38:22.3777669Z 
2023-04-29T20:38:22.3778123Z Waiting for 4 seconds.
2023-04-29T20:38:26.4273633Z 
2023-04-29T20:38:26.4274996Z SignTool Error: An error occurred while attempting to load the signing
2023-04-29T20:38:26.4275908Z 
2023-04-29T20:38:26.4277101Z 	certificate from: C:\Users\RUNNER~1\AppData\Local\Temp\code-sign-certificate-497841.pfx
2023-04-29T20:38:26.4280055Z 
2023-04-29T20:38:26.4280064Z 
2023-04-29T20:38:26.4280525Z Waiting for 5 seconds.
2023-04-29T20:38:31.4874920Z 
2023-04-29T20:38:31.4876610Z SignTool Error: An error occurred while attempting to load the signing
2023-04-29T20:38:31.4885647Z 
2023-04-29T20:38:31.4887619Z 	certificate from: C:\Users\RUNNER~1\AppData\Local\Temp\code-sign-certificate-497841.pfx
2023-04-29T20:38:31.4889149Z 
2023-04-29T20:38:31.4889357Z 
2023-04-29T20:38:31.4909907Z Waiting for 6 seconds.
2023-04-29T20:38:37.5318459Z 
2023-04-29T20:38:37.5319860Z SignTool Error: An error occurred while attempting to load the signing
2023-04-29T20:38:37.5320763Z 
2023-04-29T20:38:37.5321961Z 	certificate from: C:\Users\RUNNER~1\AppData\Local\Temp\code-sign-certificate-497841.pfx
2023-04-29T20:38:37.5322922Z 
2023-04-29T20:38:37.5322931Z 
2023-04-29T20:38:37.5323382Z Waiting for 7 seconds.
2023-04-29T20:38:44.5787417Z 
2023-04-29T20:38:44.5788750Z SignTool Error: An error occurred while attempting to load the signing
2023-04-29T20:38:44.5789736Z 
2023-04-29T20:38:44.5791104Z 	certificate from: C:\Users\RUNNER~1\AppData\Local\Temp\code-sign-certificate-497841.pfx
2023-04-29T20:38:44.5792193Z 
2023-04-29T20:38:44.5792306Z 
2023-04-29T20:38:44.5792799Z Waiting for 8 seconds.
2023-04-29T20:38:52.6246333Z 
2023-04-29T20:38:52.6247805Z SignTool Error: An error occurred while attempting to load the signing
2023-04-29T20:38:52.6251256Z 
2023-04-29T20:38:52.6252562Z 	certificate from: C:\Users\RUNNER~1\AppData\Local\Temp\code-sign-certificate-497841.pfx
2023-04-29T20:38:52.6253578Z 
2023-04-29T20:38:52.6253589Z 
2023-04-29T20:38:52.6254109Z Waiting for 9 seconds.
2023-04-29T20:39:01.6757290Z 
2023-04-29T20:39:01.6758778Z SignTool Error: An error occurred while attempting to load the signing
2023-04-29T20:39:01.6761912Z 
2023-04-29T20:39:01.6763513Z 	certificate from: C:\Users\RUNNER~1\AppData\Local\Temp\code-sign-certificate-497841.pfx
2023-04-29T20:39:01.6764704Z 
2023-04-29T20:39:01.6764820Z 
2023-04-29T20:39:01.6766157Z Removing C:\Users\RUNNER~1\AppData\Local\Temp\code-sign-certificate-497841.pfx.
2023-04-29T20:39:01.6805151Z ##[error]Action failed with error: Failed to sign 'build/ninja-multi-vcpkg/bin/RelWithDebInfo/zeal.exe'.
```

--- a/index.ts
+++ b/index.ts
@@ -8,7 +8,7 @@ import { exec } from 'child_process';
 import { env } from 'process';
 
 const asyncExec = util.promisify(exec);
-const nugetFileName = env['TEMP'] + '\\nuget.exe';
+const nugetFileName = `${env.RUNNER_TEMP}\\nuget.exe`;
 
 const timestampUrl = 'http://timestamp.digicert.com';
 
@@ -43,7 +43,7 @@ async function createCertificatePfx(): Promise<string> {
     const certificate = Buffer.from(base64Certificate, 'base64');
     if (certificate.length == 0)
         throw 'certificate value is not set.';
-    const certificateFileName = env['TEMP'] + `\\code-sign-certificate-${Math.floor((Math.random() * 1000000))}.pfx`
+    const certificateFileName = `${env.RUNNER_TEMP}\\code-sign-certificate-${Math.floor((Math.random() * 1000000))}.pfx`
     console.log(`Writing ${certificate.length} bytes to ${certificateFileName}.`);
     await fs.writeFile(certificateFileName, certificate);
     return certificateFileName;

